test/unit/cli.test.js

```diff
@@ -0,0 +1,166 @@
+      });
+    });
+
+    describe('write', () => {
+      it('should write in stream', () => {
+        cli.write('Getting commits...');
+        assert.strictEqual(stream.toString(), 'Getting commits...');
+      });
+    });
+
+    describe('log', () => {
+      it('should write in stream', () => {
+        cli.log('Getting commits...');
+        assert.strictEqual(stream.toString(), `Getting commits...${EOL}`);
+      });
+    });
+
+    describe('table', () => {
+      it('should print the first element with bold style and padding', () => {
+        cli.table('Title', 'description');
+        assert.strictEqual(stream.toString(),
+          `${chalk.bold('Title   ')}description${EOL}`);
+      });
+    });
+
+    describe('separator', () => {
+      it('should print a separator line with the specified text', () => {
+        cli.separator('Separator');
+        assert.strictEqual(
+          stream.toString(),
+          '---------------------------------- ' + chalk.bold('Separator') +
+            ' -----------------------------------' + EOL);
+      });
+
+      it('should print a separator line with a custom separator', () => {
+        cli.separator('PR', 20, '+');
+        assert.strictEqual(
+          stream.toString(),
+          '++++++++ ' + chalk.bold('PR') + ' ++++++++' + EOL);
+      });
+
+      it('should print a separator line without text', () => {
+        cli.separator();
+        assert.strictEqual(
+          stream.toString(),
+          '-------------------------------------------------------' +
+            '-------------------------' + EOL);
+      });
+    });
+
+    describe('ok', () => {
+      it('should print a success message', () => {
+        cli.ok('Perfect!');
+        assert.strictEqual(stream.toString(), `${success}  Perfect!${EOL}`);
+      });
+
+      it('should print a success message in a new line if specified', () => {
+        cli.ok('Perfect!', { newline: true });
+        assert.strictEqual(stream.toString(),
+          `${EOL}${success}  Perfect!${EOL}`);
+      });
+    });
+
+    describe('warn', () => {
+      it('should print a warning message', () => {
+        cli.warn('Warning!');
+        assert.strictEqual(stream.toString(), `${warning}  Warning!${EOL}`);
+      });
+
+      it('should print a warning message in a new line if specified', () => {
+        cli.warn('Warning!', { newline: true });
+        assert.strictEqual(stream.toString(),
+          `${EOL}${warning}  Warning!${EOL}`);
+      });
+    });
+
+    describe('info', () => {
+      it('should print an info message', () => {
+        cli.info('Info!');
+        assert.strictEqual(stream.toString(), `${info}  Info!${EOL}`);
+      });
+
+      it('should print an info message in a new line if specified', () => {
+        cli.info('Info!', { newline: true });
+        assert.strictEqual(stream.toString(), `${EOL}${info}  Info!${EOL}`);
+      });
+    });
+
+    // TODO: `Error` instance test
```

Comment:
Can you please give me an example of an`Error` instance with `data` property?

Reply:
I think you can get that by `console.log(JSON.stringify(err))` and try pulling a PR id that's actually an issue